db/migrate/20190802133303_create_triggers_backend_apis.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CreateTriggersBackendApis < ActiveRecord::Migration
+  # At this point it does not matter if it blocks these tables or not because they are not being used yet anyway
+
+  def up
+    self.class.execute_trigger_actions do |trigger|
+      connection = ActiveRecord::Base.connection
+      connection.execute(trigger.drop)
+      connection.execute(trigger.create)
```

Comment:
You can use https://github.com/3scale/porta/blob/00ce6011f14d7b1625b58f917b8afc82be259f36/lib/system/database/trigger.rb#L24-L26